rules-reviewed/eap6/uncategorized/environment-dependent.windup.xml

```diff
@@ -23,7 +23,7 @@
                 </javaclass>
             </when>
             <perform>
-                 <hint title="Dynamic class instantiation" effort="0" severity="optional">
+                 <hint title="Dynamic class instantiation" effort="1" severity="optional">
```

Comment:
We decided in the past to keep this at 0 -- if effort is attached this rule often completely takes over the story point estimates for the entire app.